app/src/components/market/common/market_top_details_open/index.tsx

```diff
@@ -19,6 +19,7 @@ const MarketTopDetailsOpen: React.FC<Props> = (props: Props) => {
   const [showingProgressBar, setShowingProgressBar] = useState(false)
 
   const { marketMakerData, title } = props
+
```

Comment:
No big deal, but in the future try to make sure you aren't committing any files that don't have real changes like this one.